File: Service/GroupMembershipManagement/Models/Setting.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

using System;

namespace Models
{

    public class Setting
    {
        public Setting()
        {
        }

        public Setting(Guid id, SettingKey settingKey, string settingValue)
        {
            Id = id;
            SettingKey = settingKey;
            SettingValue = settingValue;
        }

        public Guid Id { get; set; }
        public SettingKey SettingKey { get; set; }
        public string SettingValue { get; set; }
    }
}

File: UI/web-app/src/models/Setting.ts
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

import { SettingKey } from './SettingKey';
import { SettingName } from './SettingName';

export type Setting = {
    settingKey: SettingKey;
    settingName: SettingName;
    settingValue: string;
}